ts/src/helpers.cpp

```diff
@@ -279,12 +279,17 @@ std::map<std::string, FREQ_SETTINGS> helpers::parseFrequencies(const std::string
     std::vector<std::string> v = split(sub, ',');
     for (const std::string& xs : v) {
         std::vector<std::string> parts = split(xs.substr(1, xs.length() - 2), '|');
-        if (parts.size() == 3 || parts.size() == 4) {
+        if (parts.size() == 3 || parts.size() == 4 || parts.size() == 5) {
```

Comment:
Can parts still be 4 elements? classname and override are both part of half duplex. So 4 elements shouldn't be possible anymore,